### PR TITLE
Add /campaigns endpoint. AB-8654

### PR DESCRIPTION
**Pull request by yuriykurylenko**
Documentation for https://autobutler.atlassian.net/browse/AB-8654.

### CODE REVIEW

File: source/includes/_campaigns.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+`GET https://www.autobutler.dk/api/v2/mechanics/campaigns`
+
+
+Attribute                                                            | Type          | Can be blank? | Description
+-------------------------------------------------------------------- | ------------- | ------------- | -----------------------------------------------------------------------------
+campaigns                                                            | object        | no            | An object representing the individual conversations
+campaigns.currentCampaignPages                                       | array(object) | yes           | The array representing a list of ongoing campaigns, relevant for a mechanic
+campaigns.comingCampaignPages                                        | array(object) | yes           | The array representing a list of future campaigns, relevant for a mechanic
+campaigns.currentCampaignPages(comingCampaignPages).id               | integer       | yes           | The campaign id
```
Comment (dipth) [inline source/includes/_campaigns.md]: Please separate these into two different lines of documentation.
If you don't want to repeat yourself, then extract the content into a common JSON object and reference that instead

Comment (dipth) [inline source/includes/_campaigns.md]: Here is an example of how common json object documentation is used:
https://autobutler.github.io/mechanics_api_docs/#get-job-details

Comment (yuriykurylenko) [inline source/includes/_campaigns.md]: 👍

File: source/includes/_campaigns.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+This endpoint returns a list of campaigns relevant for the workshop.
+
+### HTTP Request
+
+`GET https://www.autobutler.dk/api/v2/mechanics/campaigns`
+
+
+Attribute                                                            | Type          | Can be blank? | Description
+-------------------------------------------------------------------- | ------------- | ------------- | ----------------------------------------------------------------------------------------------------
+campaigns                                                            | object        | no            | An object representing the individual conversations
```
Comment (dipth) [inline source/includes/_campaigns.md]: "conversations"?

Comment (yuriykurylenko) [inline source/includes/_campaigns.md]: Ups

**Pull request merged**